shared/editor/embeds/Grist.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+import Frame from "../components/Frame";
+import { EmbedProps as Props } from ".";
+
+const URL_REGEX = new RegExp(
+  "^https?://([a-z.-]+\\.)?getgrist\\.com/(.*)/p/([0-9]+)\\?embed=true$"
+);
+
+export default class Grist extends React.Component<Props> {
+  static ENABLED = [URL_REGEX];
+
+  render() {
+    return (
+      <Frame {...this.props} src={this.props.attrs.href} title="Grist Embed" />
```

Comment:
![Screenshot from 2022-08-06 00-31-46](https://user-images.githubusercontent.com/15694566/183145037-fc784a7d-6690-4340-9b3f-20bf0e53011b.png)

Just realized that one small improvement you could do here is to have a footer as shown in the sample screenshot. Use [this](https://github.com/outline/outline/blob/74d9409cc32c5923fe72e98eb961d1d56304519e/shared/editor/embeds/GoogleDocs.tsx#L13-L27) as reference.